selenium_tests/cross_listing/page_objects/cross_listing_base_page_object.py

```diff
@@ -0,0 +1,34 @@
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+
+from selenium_common.base_page_object import BasePageObject
+
+
+class Locators(object):
+    @classmethod
+    def TD_TEXT_XPATH(cls, search_text):
+        """
+        Returns a locator for the first table cell element in any table on the
+        page whose text matches `search_text`. Note: matches text of element
+        (not inclusive of child-elements) with leading and trailing space
+        stripped and whitespace normalized between words.
+        """
+        return By.XPATH, "//td[normalize-space(text())='{}']".format(
+            search_text)
+
+
+class CrossListingBasePageObject(BasePageObject):
+    def __init__(self, driver):
+        super(CrossListingBasePageObject, self).__init__(driver)
+        self.focus_on_tool_frame()
+
+    def get_cell_with_text(self, search_text):
```

Comment:
Will give it a shot to move this into base object in this round.  This is being used by course_info as well, so it's probably cleaner to move it in sooner than later.   I've only done the update to selenium_common once, so it'll be a good practice as well.